.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+      - run: npm install
+      - run: npm install
```

Comment:
@mdesmet Why are we installing npm twice?

Reply:
This is not npm install, this is installing the dependencies of the child projects.